Sources/SmileID/Classes/DocumentVerification/Model/DocumentCaptureModel.swift

```diff
@@ -132,7 +142,18 @@ class DocumentCaptureViewModel: ObservableObject, JobSubmittable, ConfirmationDi
             })
     }
 
-    func captureImage() {
+    func startManualCaptureTimer() {
+        if self.enableManualCaptureTimer == nil {
+            self.autoCaptureTimer?.invalidate()
+            self.enableManualCaptureTimer = Timer.scheduledTimer(timeInterval: 10,
+                                                                 target: self,
+                                                                 selector: #selector(manualCapture),
+                                                                 userInfo: nil,
+                                                                 repeats: false)
+        }
+    }
+
+    @objc func captureImage() {
```

Comment:
Is the `@objc` here necessary?

Reply:
Yeah it is. The `@objc` attribute is used in Swift to indicate that a method, property, or class should be available to Objective-C code. You'll notice we use the` #selector` syntax below to invoke this method. This essentially creates a reference to a method that can be called by Objective-C code. The Timer class is implemented in Objective-C and as such requires this.